app/src/main/java/org/jboss/hal/client/configuration/subsystem/transactions/TransactionView.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.hal.client.configuration.subsystem.transactions;
+
+import java.util.Map;
+import javax.annotation.PostConstruct;
+
+import com.google.common.base.Strings;
+import org.jboss.hal.ballroom.VerticalNavigation;
+import org.jboss.hal.ballroom.form.Form;
+import org.jboss.hal.ballroom.form.FormItem;
+import org.jboss.hal.ballroom.form.ValidationResult;
+import org.jboss.hal.core.mbui.MbuiContext;
+import org.jboss.hal.core.mbui.MbuiViewImpl;
+import org.jboss.hal.dmr.ModelNode;
+import org.jboss.hal.spi.MbuiElement;
+import org.jboss.hal.spi.MbuiView;
+
+import static org.jboss.hal.client.configuration.subsystem.transactions.TransactionPresenter.PROCESS_ID_SOCKET_BINDING;
+import static org.jboss.hal.client.configuration.subsystem.transactions.TransactionPresenter.PROCESS_ID_SOCKET_MAX_PORTS;
+import static org.jboss.hal.client.configuration.subsystem.transactions.TransactionPresenter.PROCESS_ID_UUID;
+
+/**
+ * @author Claudio Miranda
+ */
+@MbuiView
+@SuppressWarnings({"DuplicateStringLiteralInspection", "HardCodedStringLiteral", "WeakerAccess"})
+public abstract class TransactionView extends MbuiViewImpl<TransactionPresenter> implements TransactionPresenter.MyView {
+
+    // ------------------------------------------------------ initialization
+
+    public static TransactionView create(final MbuiContext mbuiContext) {
+        return new Mbui_TransactionView(mbuiContext);
+    }
+
+    @MbuiElement("transaction-vertical-navigation") VerticalNavigation navigation;
+    @MbuiElement("attributes-form") Form<ModelNode> attributesForm;
+    @MbuiElement("process-form") Form<ModelNode> processForm;
+    @MbuiElement("recovery-form") Form<ModelNode> recoveryForm;
+    @MbuiElement("path-form") Form<ModelNode> pathForm;
+    @MbuiElement("jdbc-form") Form<ModelNode> jdbcForm;
+
+    TransactionView(final MbuiContext mbuiContext) {
+        super(mbuiContext);
+    }
+    
+    @PostConstruct
+    void init() {
+
+        ValidationResult invalid = ValidationResult.invalid("Validation error, see error messages below.");
+        
+        // --------------- form validation for the general attributes 
+        attributesForm.addFormValidation(formItems -> {
```

Comment:
Since this is the first usage of `FormValidation` I think it makes sense to change the method signature of the validate method from `ValidationResult validate(Iterable<FormItem> formItems);` to `ValidationResult validate(Form<T> form);` and add `T` as a type parameter to `FormValidation`. 

This makes the `formItem()` helper method redundant.